examples/torch/pearl_half_cheetah_vel.py

```diff
@@ -0,0 +1,151 @@
+import akro
+import numpy as np
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.envs.env_spec import EnvSpec
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
+from garage.experiment import LocalRunner, run_experiment
+from garage.sampler import PEARLSampler
+from garage.torch.algos import PEARLSAC
+from garage.torch.embeddings import RecurrentEncoder
+from garage.torch.modules import MLPEncoder
```

Comment:
hmm why does MLPEncoder appear in `.modules` but `RecurrentEncoder` appears in `.embeddings`?